extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/runtime/PanacheQueryImpl.java

```diff
@@ -40,7 +39,6 @@
     PanacheQueryImpl(MongoCollection<? extends Entity> collection, Class<? extends Entity> entityClass, Document mongoQuery,
```

Comment:
Might as well remove the parameter if we're not using it, no?

Reply:
Yes, this was copied from Hibernate with Panache but not needed here.
I remove it (and also for the reactive implementation), a little cleanup doesn't hurt ;)